Lib/pathlib.py

```diff
@@ -521,14 +409,31 @@ def _from_parsed_parts(cls, drv, root, parts):
     @classmethod
     def _format_parsed_parts(cls, drv, root, parts):
         if drv or root:
-            return drv + root + cls._flavour.join(parts[1:])
+            return drv + root + cls._flavour.sep.join(parts[1:])
+        else:
+            return cls._flavour.sep.join(parts)
+
+    def _join_parsed_parts(self, drv2, root2, parts2):
+        """
+        Join the two paths represented by the respective
```

Comment:
```suggestion
        """Join the two paths represented by the respective
```

Could you also make the file consistent in how it formats docstrings?